sideload-repos-systemd/flatpak-create-sideload-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script is intended to be run by flatpak-sideload-usb-repo.service
+
+if ! test $# -eq 1 || ! test -d "$1"; then
+  echo "Error: first argument must be a directory"
+  exit 1
+fi
+
+# Add a link to any newly inserted drives which were copied to with "flatpak create-usb"
+for f in $1/*; do
+    if ! test -d "$f"; then
+        continue
+    fi
+    base=$(basename "$f")
```

Comment:
I wonder if just using the basename is enough? Maybe it could append some checksum of the entire path?

Reply:
Good point. I think we could just use `automount-$(sha256sum $f)`, no need to include the basename I think.